documentation/src/main/asciidoc/topics/attributes/product-attributes.adoc

```diff
@@ -1,127 +0,0 @@
-//
```

Comment:
was this removed by mistake? (just to confirm)

Reply:
it wasn't removed by mistake but I've decided to add it back without the URLs to docs. I'm going to put those downstream and copy them where needed.